akka-http-tests/src/test/scala/akka/http/scaladsl/marshalling/ContentNegotiationSpec.scala

```diff
@@ -136,6 +136,40 @@ class ContentNegotiationSpec extends FreeSpec with Matchers {
       accept(`text/plain`, `text/xml`) should select(`text/plain` withCharset `UTF-8`)
       accept(`text/xml`, `text/plain`) should select(`text/xml` withCharset `UTF-8`)
     }
+
+    /** when both acceptable media types are equally specific and have the same qValue, we choose to send the first */
+    "Accept: text/html;level=1, text/html;format=flowed" test { accept ⇒
+      accept(`text/html` withParams Map("level" → "1"), `text/html` withParams Map("format" → "flowed")) should select(`text/html` withParams Map("level" → "1") withCharset `UTF-8`)
+    }
+
+    /** when an acceptable media type is more specific, it is matched even when it has a lower qValue */
```

Comment:
Or maybe just the description is wrong?

Reading through the description in https://tools.ietf.org/html/rfc7231#section-5.3.2, precedence for the `Accept` header would be this (most specific first):

 * `text/html;format=flowed;q=0.5`
 * `text/html;q=1`

Then for the available content-type for this resource the table would look like this:

| Content Type | Matched Range | qValue |
| - | - | - |
| `text/html;level=1` | `text/html;q=1` | `1` |
| `text/html;format=flowed` |  `text/html;format=flowed;q=0.5` | `0.5` |

So, the result makes sense here (but not the comment because in this case the less specific range was chosen).